app/endpoints/messages.py

```
@@ -268,8 +284,35 @@ async def create_conversation_with_first_message(
 
 
 
-@router.get("/conversation/{conversation_id}", response_model=list[Message])
-def read_messages_by_conversation(conversation_id: str, db: Session = Depends(get_db)):
-    messages = get_messages_by_conversation(db=db, conversation_id=conversation_id)
-    return messages
+@router.get("/conversation/{conversation_id}")
+def read_messages_by_conversation(
+    conversation_id: str,
+    db: Session = Depends(get_db),
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)